contracts/feirari/RewardsDistributorAdmin.sol

```diff
@@ -2,21 +2,33 @@ pragma solidity ^0.8.0;
 
 import "../refs/CoreRef.sol";
 import "./IRewardsDistributorAdmin.sol";
+import "@openzeppelin/contracts/access/AccessControlEnumerable.sol";
+
+contract RewardsDistributorAdmin is IRewardsDistributorAdmin, CoreRef, AccessControlEnumerable {
+
+    bytes32 public constant override AUTO_REWARDS_DISTRIBUTOR = keccak256("AUTO_REWARDS_DISTRIBUTOR");
 
-contract RewardsDistributorAdmin is IRewardsDistributorAdmin, CoreRef {
     /// @notice rewards distributor contract
     IRewardsDistributorAdmin public rewardsDistributorContract;
 
     /// @param coreAddress address of core contract
     /// @param _rewardsDistributorContract admin rewards distributor contract
     constructor(
         address coreAddress,
-        IRewardsDistributorAdmin _rewardsDistributorContract
+        IRewardsDistributorAdmin _rewardsDistributorContract,
+        address[] memory _autoRewardDistributors
     ) CoreRef(coreAddress) {
         rewardsDistributorContract = _rewardsDistributorContract;
         /// @notice Governance should create new admin role for this contract
         /// and then grant this role to all AutoRewardsDistributors so that they can call this contract
-        _setContractAdminRole(keccak256("REWARDS_DISTRIBUTOR_ADMIN_ROLE"));
+        _setContractAdminRole(keccak256("TRIBAL_CHIEF_ADMIN_ROLE"));
+        /// assign the admin role of AUTO_REWARDS_DISTRIBUTOR to DEFAULT_ADMIN_ROLE
```

Comment:
comments that don't provide additional context (like this one) can be taken out

Reply:
More useful would be explaining that there are two different access control systems here, one in Core and one locally. Explaining the roles and relationships probably in a top level block comment above the contract definition will be ideal

Reply:
Removed useless comment and added context comment to top of file